docs/configuration-reference/components/aws-ebs-csi-driver.md

```diff
@@ -0,0 +1,47 @@
+# CSI Driver for Amazon EBS configuration reference for Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Applying](#applying)
+* [Destroying](#destroying)
+
+## Introduction
+
+The [CSI Driver for Amazon EBS](https://github.com/kubernetes-sigs/aws-ebs-csi-driver)
+provides a CSI interface used by Container Orchestrators to manage the lifecycle
+of Amazon EBS volumes.
+
+## Prerequisites
+
+* A Lokomotive cluster accessible via `kubectl` deployed on Packet.
+
+## Configuration
+
+The CSI Driver component takes no confirguration arguments.
```

Comment:
I think we should have a knob for this component, which enables/disables the default storage class. This way, the user can still use this component, but let's say have OpenEBS as a default storage option.

Reply:
How do configuration knobs work here? I added back `chartValuesTmpl`, hooked up the component struct, and added a boolean to `values.yaml`. I also wrapped the storageclass code in an `if` statement, but according to `lokoctl component render-manifest` it's still getting created even with my option set to false.

Reply:
Hm, did you push the new code? I cannot see it, so it's hard to tell why is it like this. According to your description, everything seems correct.

Reply:
Pushed a WIP commit.